org-formation/300-account-defaults/bedrock-agent-role.yaml

```diff
@@ -18,6 +18,8 @@ Resources:
                 aws:SourceAccount: !Ref AWS::AccountId
               ArnLike:
                 aws:SourceArn: !Sub "arn:aws:bedrock:${AWS::Region}:${AWS::AccountId}:agent/*"
+      ManagedPolicyArns:
+        - arn:aws:iam::aws:policy/AmazonS3FullAccess
```

Comment:
you can also replace your `bedrockAgentPolicy` with the AWS managed `AWSLambdaRole` policy..
![image](https://github.com/user-attachments/assets/609af18a-0a19-4dbf-b469-b3b6978a34d3)